string type
string index
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
int8[4] buttons